src/user-statuses/userStatusesModel.js

```diff
@@ -22,6 +21,8 @@ import { kUserStatusZero } from './userStatusesCore';
 // Selectors.
 //
 
+export const getUserStatuses = (state: PerAccountState): UserStatusesState => state.userStatuses;
```

Comment:
And unexport it -- we want consumers of the model to use the

Did you mean to delete the `export` here?

Reply:
Ah, yep! Thanks!